pkg/apic/definitions.go

```diff
@@ -142,3 +146,40 @@ type PlatformTeam struct {
 	Name    string `json:"name"`
 	Default bool   `json:"default"`
 }
+
+type oas2Swagger struct {
+	openapi2.Swagger
+}
+
```

Comment:
We decided previously to not put code into the definitions.go file. These files are being excluded from sonar scans.

Reply:
I will move it